src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -1,34 +1,45 @@
 package tech.cryptonomic.conseil
 
 import akka.actor.ActorSystem
-import akka.Done
-import mouse.any._
 import com.typesafe.scalalogging.LazyLogging
 import tech.cryptonomic.conseil.tezos.{
-  TezosTypes,
+  AccountsDataFetchers,
+  ApiOperations,
+  BlocksDataFetchers,
   FeeOperations,
-  ShutdownComplete,
+  NodeOperator,
+  TezosDatabaseOperations => TezosDb,
   TezosErrors,
-  TezosNodeInterface,
-  TezosNodeOperator,
-  TezosDatabaseOperations => TezosDb
+  TezosTypes
 }
-import tech.cryptonomic.conseil.tezos.TezosTypes.{Account, AccountId, BlockTagged, Delegate, PublicKeyHash}
+import tech.cryptonomic.conseil.tezos.TezosRemoteInstances.Akka.{ShutdownComplete, TezosNodeContext}
+import tech.cryptonomic.conseil.tezos.TezosTypes._
 import tech.cryptonomic.conseil.io.MainOutputs.LorreOutput
 import tech.cryptonomic.conseil.util.DatabaseUtil
+import tech.cryptonomic.conseil.util.EffectsUtil.{runDbIO, toIO}
 import tech.cryptonomic.conseil.config.{Custom, Everything, LorreAppConfig, Newest}
 import tech.cryptonomic.conseil.config.Platforms
 
 import scala.concurrent.duration._
 import scala.annotation.tailrec
-import scala.concurrent.{Await, Future}
+import scala.concurrent.Await
 import scala.concurrent.duration.Duration
 import scala.util.{Failure, Success, Try}
+import scala.{Stream => _}
+
+import cats.effect.{ContextShift, IO}
+import scala.util.control.NonFatal
+import slick.dbio.DBIO
 
 /**
   * Entry point for synchronizing data between the Tezos blockchain and the Conseil database.
   */
 object Lorre extends App with TezosErrors with LazyLogging with LorreAppConfig with LorreOutput {
+  import cats.instances.list._
```

Comment:
This object needs a cleanup. I'm ok with having mainLoop here, but in my opinion, rest should be in the other class/object. At the moment it's hard to see where this application starts. Let's keep `Lorre` main object reasonably small. What do you think?

Reply:
I'm all for it, I actually mention this in the lengthy description of the PR above.

Lorre's class is quite a mess now, but I'd prefer to leave the refactor for a later PR, as this is already so packed-up!